packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1058,4 +1060,111 @@ module.exports.external = [
       return PERSONAL_RELATIONSHIP_TYPE_CODE;
     },
   },
+  {
+    name: "decodeHtmlEntities",
+    description: "Decodes html strings",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return "";
+      }
+      const result = he.decode(str);
+      return result;
+    },
+  },
+  {
+    name: "removeNonNumeric",
+    description: "Removes all non-numeric characters from a string, keeping decimal points",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return "";
+      }
+      return str.replace(/[^0-9.]/g, '');
+    },
+  },
+  {
+    name: "convertFeetAndInchesToCm",
+    description: "Checks if a string is in the format 'number ft number in' and if so, converts the feet and inches to centimeters",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return { isValid: false };
+      }
+      const match = str.match(/^(\d+(?:\.\d+)?) ft (\d+(?:\.\d+)?)( in)?$/);
+      if (match) {
+        const feet = parseFloat(match[1]);
+        const inches = parseFloat(match[2]);
+        const cm = (feet * 12 + inches) * 2.54;
+        return { isValid: true, value: cm, unit: 'cm' };
+      } else {
+        return { isValid: false };
+      }
+    },
+  },
+  {
+    name: "extractNumberAndUnit",
+    description: "Checks if a string is in the format 'number unit' and if so, extracts the number and the unit",
```

Comment:
we sometimes get the unit in the value instead of separate in the unit field. If we have a numeric value, followed by an alphabetical or parenthesis, then we separate them and treat the first part as the value and the second as a unit. This is a pretty lenient rule
Units sometimes look like this: mg/dL or (mm)